testenv/suite_test.go

```diff
@@ -59,9 +58,8 @@ func init() {
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
 
-	RunSpecsWithDefaultAndCustomReporters(t,
-		"Controller Suite",
-		[]Reporter{printer.NewlineReporter{}})
+	RunSpecs(t,
```

Comment:
is this related to this PR (upgrade ? )

Reply:
Yes
RunSpecsWithDefaultAndCustomReporters is deprecated replace by RunSpecs.